sass/_emoji.scss

```diff
@@ -119,18 +126,15 @@
                     flex-wrap: wrap;
 
                     .emoji-category {
-                        padding: 0.25em 0;
                         font-size: var(--font-size-huge);
                         &.picked {
                             background-color: white;
                             border: 1px var(--chat-head-color) solid;
                             border-bottom: none;
                         }
-                        &.selected a, &:hover a {
-                            background-color: var(--highlight-color-darker);
-                        }
```

Comment:
Why did you remove this? The hover effect is now gone for the emoji categories.

Reply:
It seems like my cache was showing me a different version where a hover still showed, the intention was to make a visual improvement. I will put that back.